make/istio/run

```diff
@@ -16,24 +16,68 @@ if [ ! -d "$istio_path" ]; then
     | tar zx -C "${istio_path}" --strip-components=1
 fi
 
+istio_init_chart="${istio_path}/install/kubernetes/helm/istio-init"
+istio_chart="${istio_path}/install/kubernetes/helm/istio"
 
-chart="${istio_path}/install/kubernetes/helm/istio"
+# Install all the Istio Custom Resource Definitions.
+istio_init_args=(
+  "${ISTIO_INIT_HELM_RELEASE}"
+  --namespace "${ISTIO_NAMESPACE}"
+  --timeout 900 
+)
+helm upgrade --install "${istio_init_args[@]}" "${istio_init_chart}"
+
+max_attempts=5
+for (( i=1; i<=max_attempts; i++ ))
```

Comment:
I think if you pass `--wait` to `helm upgrade` you don't need this whole big block for waiting.